main/menu/menuManiaRenderer.h

```diff
@@ -0,0 +1,49 @@
+/*! \file menuManiaRenderer.h
+ *
+ * \section menuManiaRenderer_design Design Philosophy
+ *
+ * A menu renderer takes the data in the menu_t data structure and renders it to the display. The menu data structure is
+ * separated from the rendering logic so that it is possible to draw menus with differing themes.
+ *
+ * \section menuManiaRenderer_usage Usage
+ *
+ * First a ::menu_t data structure must be created. For that usage, see menu.h.
+ *
+ * The menu renderer is initialized with initMenuManiaRenderer() and deinitialized with deinitMenuManiaRenderer().
+ * Menu renderers must be deinitialized or they will leak memory.
+ * When initializing the menu, a font must be passed in as an argument. This renderer will not allocate its own font to
+ * avoid allocating the same font twice for a given mode (once for the menu and again for the mode itself).
+ *
+ * The menu is drawn with drawMenuMania(). This will both draw over the entire display and light LEDs. The menu may be
+ * drawn on top of later.
+ *
+ * \section menuManiaRenderer_example Example
+ *
+ * See menu.h for examples on how to use menuManiaRenderer
+ */
+
+#ifndef _MENU_LOGBOOK_RENDERER_RENDERER_H_
```

Comment:
Gotta update that guard!

Reply:
lol, my search must have been case sensitive